jekyll/_cci2/executor-types.md

```diff
@@ -64,43 +72,69 @@ More details on the Docker Executor are available in the [Configuring CircleCI](
 
 ## Using Machine
 
-The `machine` option will run your jobs in a dedicated, ephemeral Virtual Machine (VM) with the following technical specifications:
+The `machine` option runs your jobs in a dedicated, ephemeral VM
+that has the following specifications:
 
 CPUs | Processor                 | RAM | HD
 -----|---------------------------|------------
 2    | Intel(R) Xeon(R) @ 2.3GHz | 8GB | 100GB
 {: class="table table-striped"}
 
-**Note**: Use of `machine` may require additional fees in a future pricing update. 
+Using the `machine` executor
+gives your application full access to OS resources
+and provides you with full control over the job environment.
+This control can be useful in situations
+where you need to use `ping`
+or modify the system with `sysctl` commands.
+
+Using the `machine` executor also enables you
+to build a Docker image
+without downloading additional packages
+for languages like Ruby and PHP.
 
-To use the machine executor with the default `machine` image, set the `machine` key to `true` in `.circleci/config.yml`:
+**Note**:
+Using `machine` may require additional fees in a future pricing update.
+
+To use the machine executor,
+set the `machine` key to `true` in `.circleci/config.yml`:
 
 ```yaml
+version: 2
 jobs:
   build:
     machine: true
 ```
 
-Using the `machine` executor enables your application with full access to OS resources and provides you with full control over the job environment, if for example, you need to use `ping` or to modify system with `sysctl` commands. In addition, it enables your repo to build a docker image without additional downloads for languages like Ruby and PHP.
+The default image for the machine executor is `circleci/classic:latest`.
+You can specify other images
+by using the `image` key.
 
-This example specifies a particular image for the `machine` executor:
+**Note:**
+The `image` key is not supported on private installations of CircleCI.
+See the [VM Service documentation]({{ site.baseurl }}/2.0/vm-service) for more information.
 
 ```yaml
+version: 2
 jobs:
   build:
     machine:
-      image: circleci/classic:201708-01
-``` 
-
-Following are the available machine images:
+      image: circleci/classic:2017-01  # pins image to specific version using YYYY-MM format
+```
 
-* `circleci/classic:latest` is the default image. Changes to this will be announced at least one week before they go live.
-* `circleci/classic:edge` receives the latest updates and will be upgraded at short notice.
-* `circleci/classic:[year-month]` This lets you pin the image version to prevent breaking changes. Refer to the [Configuring CircleCI](https://circleci.com/docs/2.0/configuration-reference/#machine) document for versions.
```

Comment:
Not sure why we should remove link to the config ref here? Otherwise these minor changes are fine with me, thanks!

Reply:
@michelle-luna good question. The configuration reference doesn't give any information about versions. If we're going to link to the Configuration Reference anywhere, I think it might make more sense at the top of this `machine` key section.